package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "image-rendering",
+  "version": "0.1.0",
+  "description": "Handles parsing images from CAPI and rendering them in the *-rendering projects",
+  "main": "index.ts",
+  "dependencies": {
+    "@emotion/core": "^10.0.28",
+    "@guardian/src-foundations": "^2.0.0",
+    "@guardian/types": "github:guardian/types",
```

Comment:
semver versioning like you added to apps-rendering?

Reply:
Good idea!